bindata/etcd/quorum-restore-pod.yaml

```diff
@@ -3,6 +3,9 @@ kind: Pod
 metadata:
   name: etcd
   namespace: openshift-etcd
+  # we set a static UUID here to tell kubelet that this is different pod to what's usually running
+  # this ensures that the pod status is correctly updated
+  uid: 55aed6ab-58f4-4ae2-bf1c-54be36669b8a
```

Comment:
@rphillips this seems to still trigger the same kind of "no status update" message:

> Oct 15 10:30:03 ci-ln-xqds2qb-72292-knc4z-master-2 kubenswrapper[129172]: I1015 10:30:03.475366  129172 status_manager.go:861] "Pod was deleted and then recreated, skipping status update" pod="openshift-etcd/etcd-ci-ln-xqds2qb-72292-knc4z-master-2" oldPodUID="55aed6ab-58f4-4ae2-bf1c-54be36669b8a" podUID="4a473f662e91db7e22a95e1cf4e4e8cd"

here's the full grep for the UID

[log.txt](https://github.com/user-attachments/files/17376738/log.txt)